test/dommock/DOMMockBuilder.ts

```diff
@@ -347,7 +347,9 @@ export default class DOMMockBuilder {
               // @ts-ignore
               mediaStreamTrack.kind = 'video';
             }
-            mediaStream.addTrack(mediaStreamTrack);
+            if (!mockBehavior.getDisplayMediaWithSourceId) {
+              mediaStream.addTrack(mediaStreamTrack);
```

Comment:
Why do we need to do this? mediaStream from getUserMedia should have tracks.

Reply:
Synced with you offline. Fixing the DOMMockBuilder for this.